### PR TITLE
task 12 - admin page

### PR DESCRIPTION
**Pull request by rohannbajpai**
Adds fontawesome icon support
Screen Shot 2023-10-29 at 5 41 36 PM
Screen Shot 2023-10-29 at 5 41 59 PM

### CODE REVIEW

File: jaba/src/pages/SharedComponents/Paginator.tsx
Comment (yieb) [inline jaba/src/pages/SharedComponents/Paginator.tsx:1]: I love that you're organizing your code well, but it seems like you have styling for these shared components in the same CSS file that styles only the AdminView tsx files. Could you take the CSS styling related to the Paginator and SearchBar and put those in their own CSS module files along with Paginator and Searchbar in their own subdirectories under the components folder

**Comment by yieb** [comment]
Could you change the icons you used on the popup for each user to be the icons that the designer use on the Figma design?
image

**Comment by yieb** [comment]
From the screenshot in your PR, it looks like your custom fonts are also not working properly?

**Review by RickyLi15 — COMMENTED** [review]
Please look at the Admin "users" page on Sigma, some of your stylings are off, like the background color, buttons, etc. Also, for uniformity can you use CSS modules instead? When importing styles from your CSS module file, please make sure to use the format "import styles from .....".

**Comment by yieb** [comment]
image

Can you also update how the pagination selection looks like to match the Figma

**Comment by yieb** [comment]
Also, please merge main into your branch to get the custom fonts stored in the fonts/ folder and update the CSS to use the fonts that the designers are using.

If you want to use the DM Serif Text font, add `font-family: 'DM Serif Text'` to the CSS file, and if you want to use Inter, add `font-family: 'Inter'`

**Review by RickyLi15 — APPROVED** [review]

**Pull request merged**